pallets/identity/src/auth.rs

```diff
@@ -157,6 +158,8 @@ impl<T: Config> Module<T> {
         auth_id: &u64,
     ) -> Option<Authorization<T::AccountId, T::Moment>> {
         Self::authorizations(target, *auth_id).filter(|auth| {
+            // check if count is zero
+            auth.count.eq(&0u32);
             auth.expiry
```

Comment:
The return value from `auth.count.eq(0u32)` isn't being used here.

This closure needs to return `true` if the count is zero OR the auth has expired.